spikesorters/yass/yass.py

```diff
@@ -0,0 +1,210 @@
+import copy
+from pathlib import Path
+import os
+import numpy as np
+from numpy.lib.format import open_memmap
+import sys
+
+import spikeextractors as se
+from ..basesorter import BaseSorter
+from ..utils.shellscript import ShellScript
+from ..sorter_tools import recover_recording
+
+try:
+    import yaml
+    import yass
+    HAVE_YASS = True
+except ImportError:
+    HAVE_YASS = False
+
+
+class YassSorter(BaseSorter):
+    """
+    """
+
+    sorter_name = 'yass'
+    requires_locations = False
+
+    _default_params = {
+        }
+
+    _params_description = {
+    }
+
+    sorter_description = """Yass description; link to biorxiv"""
+
+    installation_mesg = """\nTo use Yass run:\n
+        >>> pip install yass-algorithm
+
+        More information ...
+    """
```

Comment:
@catubc can you fill this in?

Reply:
I added more descripotion.  More to follow